vim/plugin-managers.md

```diff
@@ -0,0 +1,39 @@
+# Plugin Managers
+
+Today I learned that vim has two main plugin managers:
+
+1. Tim Pope<sup>1</sup>'s [Pathogen][1]
+1. gmarik's [Vundle][2]
+
+Basically, the only difference is in the way each installs
+the plugins. You can install plugins manually without either manager.
+
+The biggest difference is that with Pathogen, you have to clone
+and install plugins on the filesystem manually. With Vundle,
+you add directives to your `~/.vimrc` and run a command
+to install/update the plugins, thus abstracting vim's away
+how vim's plugin system works.
+
+Both plugin managers are also plugins. Yeah.
+
+**TDATIL (The Day After Today I Learned)**
```

Comment:
I don't think this section is necessary, I think it would make more sense to put all of these together. Up to you though.